File: URI Online Judg/Beginner/URI-C-Plus-Plus/1015.cpp
#include <iostream>
#include <math.h>
#include <stdio.h>

using namespace std;

int main(int argc, char const *argv[]) {
	double x1, y1, x2, y2, result;

	std::cin >> x1 >> y1;
	std::cin >> x2 >> y2;

	result = sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2));
	printf("%.4f\n", result );
	return 0;
}

File: URI Online Judg/Beginner/URI-Python/1015.py
import math

num = input().split(' ')

x1, y1 = num

num = input().split(' ')

x2, y2 = num

x1, y1, x2, y2 = float(x1), float(y1), float(x2), float(y2)

num = math.sqrt(math.pow((x2 - x1), 2) + math.pow((y2 - y1), 2))

print("%.4f"%num)
